ls1intum/Artemis
# Bugfix/course statistics

### Checklist

### Motivation and Context
The calculation of the course statistics are not working as intended right now. Some exercises are wrongly listed.

### Description
Now only exercises are shown where the due date is over. Exercises with pending results are marked as so.

### Steps for Testing

1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Create 4 modeling exercises
4. Participate in one before the due date and grade the submission
5. Participate in one before the due date and don't grade the submission
6. Participate in one after the due date
7. Don't participate in one.
8. After all due dates go to the statistics
9. all 4 exercises should be shown and the tooltips should reflect their status


### Screenshots
![image](https://user-images.githubusercontent.com/10998002/59684942-59aebd00-91db-11e9-8cf2-633d9dd8b16a.png)
